props/BC_Working_temp_relevant_only.h

```diff
@@ -28,6 +28,105 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   SaberBCButtons() : PropBase() {}
   const char* name() override { return "SaberBCButtons"; }
 
+BLADE_ID_CLASS_INTERNAL blade_id;  // still needed?
```

Comment:
What is this for?

Reply:
Perfect question, and hence the comment to myself. Searching The Crucible for it, it looks like it's a remnant from the first attempt of implementing constant BladeID monitoring.
Gone now.